string tag_id
float64 dist_t2a0
float64 dist_t2a1
float64 dist_t2a2
float64 dist_t2a3